packages/medusa/src/api/auth/[actor_type]/[auth_provider]/callback/route.ts

```diff
@@ -14,10 +14,11 @@ export const GET = async (req: MedusaRequest, res: MedusaResponse) => {
     ContainerRegistrationKeys.CONFIG_MODULE
   )
 
-  const authMethods = (config.projectConfig?.http as any)?.authMethods ?? {}
+  const authMethodsPerActor =
+    config.projectConfig?.http?.authMethodsPerActor ?? {}
```

Comment:
Okay. Looks like we can update the `ConfigModule` type to make it optional.

Reply:
@thetutlage yeah it should be optional